build/commands/lib/config.js

```diff
@@ -1253,7 +1254,6 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
     }
 
     if (this.getCachePath()) {
-      console.log("using git cache path " + this.getCachePath())
```

Comment:
this log appears each time `Config` instance is created (we do create it each time we access `config.defaultConfig`).